src/gpu/nvidia/cudnn_binary.cpp

```diff
@@ -19,6 +19,9 @@
 #include "gpu/nvidia/sycl_cuda_scoped_context.hpp"
 #include "gpu/nvidia/sycl_cuda_stream.hpp"
 #include "sycl/sycl_buffer_memory_storage.hpp"
+#include "sycl_cuda_helper.hpp"
```

Comment:
Nit: proposing to name it `sycl_cuda_memory_storage.hpp`.

Reply:
I think that could be confusing with other memory storage types. I am renaming it instead to `sycl_cuda_memory_storage_helper.hpp`